class Solution {
    public int LastIndex(String s, char c) {
        for (int i = s.length() - 1; i >= 0; i--)
            if (s.charAt(i) == c)
                return i;
        return -1;
    }
}